recipes/bacnet-stack/config.yml

```
@@ -0,0 +1,4 @@
+---
+versions:
+  "0.8.6":
```

```suggestion
  "20200218":
```
The version inside the `config.yml` and `conandata.yml` must match.